packages/react-positioning/src/createArrowStyles.ts

```diff
@@ -26,7 +34,9 @@ import type { Theme } from '@fluentui/react-theme';
  *
  * @param size - dimensions of the square arrow element in pixels.
  */
-export function createArrowStyles(size?: number): MakeStylesStyleRule<Theme> {
+export function createArrowStyles(options?: ArrowStylesOptions): MakeStylesStyleRule<Theme> {
+  const size = options?.size;
+  const dir = options?.dir ?? 'ltr';
```

Comment:
nit: might be cleaner ? 

```suggestion
export function createArrowStyles(options: ArrowStylesOptions = {}): MakeStylesStyleRule<Theme> {
  const { size, dir = ' ltr' } = options; 
```